src/main/java/Hsien.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Hsien {
+
+    public static void printLogo() {
+        System.out.println(" _   _         _____        _ _   _ ");
+        System.out.println("| | | | ##### |_   _|##### |   \\ | |");
+        System.out.println("| |_| |#        | |  #     | |\\ \\| |");
+        System.out.println("|  _  | #####   | |  ##### | | \\ \\ |");
+        System.out.println("| | | |      # _| |_ #     | |  \\  |");
+        System.out.println("|_| |_| ##### |_____|##### |_|   \\_|\n");
+    }
+
+    public static void printLine() {
+        System.out.println("\n" + "-".repeat(50) + "\n");
+    }
+
+    public static void printList(ArrayList<Task> tasks) {
+        int counter = 1;
+        System.out.println("Here are the tasks in your list!");
+        for (Task task : tasks) {
+            System.out.printf("%s. %s%n", counter, task.getStatusDescription());
+            counter += 1;
+        }
+    }
+
+    public static void main(String[] args) {
+        printLine();
+        printLogo();
+        // Greet
+        System.out.println("Hello! I am Hsien, your personal chatbot");
+        printLine();
+
+        ArrayList<Task> messages = new ArrayList<>();
```

Comment:
instead of messages could be names tasks or tasklist because u are using an arraylist, just for better clarity